pallets/livingassets-ownership/src/lib.rs

```diff
@@ -63,17 +65,26 @@ pub mod pallet {
         #[pallet::weight(10_000 + T::DbWeight::get().writes(1).ref_time())] // TODO set proper weight
         pub fn create_collection(origin: OriginFor<T>, collection_id: u64) -> DispatchResult {
             let who = ensure_signed(origin)?;
+            Self::do_create_collection(collection_id, who)
+        }
+    }
 
-            ensure!(
-                !OwnerOfCollection::<T>::contains_key(collection_id),
-                Error::<T>::CollectionAlreadyExists
-            );
+    /// LivingAssetsOwnership trait:w
```

Comment:
I think you should move this to outside the `mod pallet`. Implementation as well.